src/app/pages/questions/components/question/text-input/text-input.component.html

```diff
@@ -29,10 +29,12 @@
   <ion-item>
     <ion-input
       type="text"
+      enterkeyhint="go"
       placeholder="{{ 'PLACEHOLDER_TEXT_INPUT' | translate }}"
       (ionChange)="emitAnswer($event.value)"
-      (ionFocus)="emitTextInputFocus(true)"
-      (ionBlur)="emitTextInputFocus(false)"
+      (ionFocus)="emitKeyboardEvent('focus')"
+      (ionBlur)="emitKeyboardEvent('blur')"
+      (keyup.enter)="emitKeyboardEvent('enter')"
```

Comment:
Can you edit the key name on the keyboard. I have seen this in some apps where Go is shown or Search is shown in the keyboard. Can we have "Next" shown in the keyboard (or Finish if it's the last question). Not urgent though, can be added as an issue to be tackled later.